integration_tests/tests/manage/booking.cy.ts

```diff
@@ -141,6 +145,55 @@ context('Booking', () => {
 
       expect(requestBody.arrivalDate).equal(booking.arrivalDate)
       expect(requestBody.departureDate).equal(booking.departureDate)
+      expect(requestBody.eventNumber).equal(offences[0].deliusEventNumber)
+    })
+  })
+
+  it('should allow me to select an index offence if there is more than one', () => {
+    // Given I am signed in as a workflow manager
+    signIn(['workflow_manager'])
+
+    const multipleOffences = activeOffenceFactory.buildList(3)
+
+    cy.task('stubBookingCreate', { premisesId: premises.id, booking })
+    cy.task('stubPremisesSummary', premises)
+    cy.task('stubFindPerson', { person })
+    cy.task('stubPersonOffences', { person, offences: multipleOffences })
+
+    // Given I visit the first new booking page
+    const bookingNewPage = BookingFindPage.visit(premises.id, booking.bed.id)
+
+    // When I enter the CRN to the form
+    bookingNewPage.enterCrn(person.crn)
+    bookingNewPage.clickSubmit()
+
+    // Then I should be redirected to the second new booking page
+    Page.verifyOnPage(BookingNewPage)
+    const bookingCreatePage = new BookingNewPage(premises.id)
+    bookingCreatePage.verifyPersonIsVisible(person)
+
+    // And the index offence radio buttons should be visible
+    bookingCreatePage.shouldShowOffences(multipleOffences)
+
+    // Given I have entered a CRN and the person has been found
+    // When I fill in the booking form
+    bookingCreatePage.completeForm(booking)
+    bookingCreatePage.selectOffence(multipleOffences[1])
+    bookingCreatePage.clickSubmit()
+
+    // Then I should be redirected to the confirmation page
+    Page.verifyOnPage(BookingConfirmation)
+    const bookingConfirmationPage = new BookingConfirmation()
+    bookingConfirmationPage.verifyBookingIsVisible(booking)
+
+    // And the booking should be created in the API
+    cy.task('verifyBookingCreate', { premisesId: premises.id }).then(requests => {
+      expect(requests).to.have.length(1)
+      const requestBody = JSON.parse(requests[0].body)
+
+      expect(requestBody.arrivalDate).equal(booking.arrivalDate)
+      expect(requestBody.departureDate).equal(booking.departureDate)
+      expect(requestBody.eventNumber).equal(multipleOffences[1].deliusEventNumber)
```

Comment:
I think it would be worth updating the Booking error test below this and the stub to include a missing `eventNumber`

Reply:
Good plan!